core/src/modules/ops/record.ts

```diff
@@ -207,6 +210,14 @@ export namespace RecordOps {
       ands.push({ modelId });
     }
 
+    // are we looking for invalid records
+    if (state === "invalid") {
+      countRequiresIncludes = true;
```

Comment:
is it required?  Wouldn't this be like the state filter?

Reply:
Record state isn't being set to invalid, a top level flag is.